src/main/java/org/sagebionetworks/bridge/spring/controllers/AssessmentController.java

```diff
@@ -0,0 +1,254 @@
+import static org.sagebionetworks.bridge.Roles.ADMIN;
+import static org.sagebionetworks.bridge.Roles.DEVELOPER;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import org.sagebionetworks.bridge.BridgeUtils;
+import org.sagebionetworks.bridge.exceptions.UnauthorizedException;
+import org.sagebionetworks.bridge.models.PagedResourceList;
+import org.sagebionetworks.bridge.models.StatusMessage;
+import org.sagebionetworks.bridge.models.accounts.UserSession;
+import org.sagebionetworks.bridge.models.assessments.Assessment;
+import org.sagebionetworks.bridge.models.assessments.AssessmentDto;
+import org.sagebionetworks.bridge.services.AssessmentService;
+
+@CrossOrigin
+@RestController
+public class AssessmentController extends BaseController {
+    static final String SHARED_ASSESSMENTS_ERROR = "Only shared assessment APIs are enabled for the shared assessment library.";
+    
+    private AssessmentService service;
+    
+    @Autowired
+    final void setAssessmentService(AssessmentService service) {
+        this.service = service;
+    }
+    
+    @GetMapping("/v1/assessments")
+    public PagedResourceList<AssessmentDto> getAssessments(@RequestParam(required = false) String offsetBy,
+            @RequestParam(required = false) String pageSize,
+            @RequestParam(name = "cat", required = false) Set<String> categories,
+            @RequestParam(name = "tag", required = false) Set<String> tags,
+            @RequestParam(required = false) String includeDeleted) {
+        UserSession session = getAuthenticatedSession(DEVELOPER);
+        
+        String appId = session.getStudyIdentifier().getIdentifier();
+        if (SHARED_STUDY_ID_STRING.equals(appId)) {
+            throw new UnauthorizedException(SHARED_ASSESSMENTS_ERROR);
+        }
+        
+        int offsetByInt = BridgeUtils.getIntOrDefault(offsetBy, 0);
+        int pageSizeInt = BridgeUtils.getIntOrDefault(pageSize, API_DEFAULT_PAGE_SIZE);
+        boolean incDeletedBool = Boolean.valueOf(includeDeleted);
+        
+        PagedResourceList<Assessment> page = service.getAssessments(
+                appId, offsetByInt, pageSizeInt, categories, tags, incDeletedBool);
+        
+        List<AssessmentDto> dtos = page.getItems().stream()
+                .map(assessment -> AssessmentDto.create(assessment))
```

Comment:
As discussed, I've moved the managed object boundary into the DAO itself, so that object is now called HibernateAssessment, and Assessment is the object as passed into the system (what was the DTO).